src/js/components/MapPanelLocationBar.jsx

```diff
@@ -477,7 +471,7 @@ export default class MapPanelLocationBar extends React.Component {
                     renderSuggestion={this.renderSuggestion}
                     onSuggestionSelected={this.onSuggestionSelected}
                     inputProps={inputProps}
-                    focusFirstSuggestion={false}
+                    focusFirstSuggestion={true}
```

Comment:
Oh derp! You are right. I thought this was a cool feature to have but it's only true if we didn't search. My bad!